### PR TITLE
Cron job auto builder adjustments and third page UI

### PR DESCRIPTION
**Pull request by anooshrees**

### CODE REVIEW

**Comment by anooshrees** [comment]
@rogerlaucorp @julian-yang closed this earlier today but am ready for review now!

File: src/main/webapp/script.js [hunk 1]
```diff
@@ -87,6 +87,18 @@ function addQueryParameter(key, value){
...
 
+function logAddressInput(){
+  let streetNumber = document.getElementById('street_number').value;
+  let route = document.getElementById('route').value;
+  let city = document.getElementById('locality').value;
+  let state = document.getElementById('administrative_area_level_1').value;
+  let zipCode = document.getElementById('postal_code').value;
+  let country = document.getElementById('country').value;
+
+  addQueryParameter("address", streetNumber + " " + route + " " + city + " "
```
Comment (julian-yang) [inline src/main/webapp/script.js:98]: Please use template literals
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Comment (anooshrees) [reply src/main/webapp/script.js:98]: I didn't know this was an option -- thanks for linking! Changing right now.

Comment (anooshrees) [reply src/main/webapp/script.js]: @julian-yang need your approval to push, thanks!

**Pull request merged**